tlf/test_common.go

```diff
@@ -16,6 +16,17 @@ func FakeID(b byte, public bool) ID {
 	return ID{bytes}
 }
 
+// FakeIDRandomOrBust creates a fake public or private TLF ID from the given
+// byte, and fill the rest with empty bytes.
```

Comment:
I just didn't want to change all the signatures. I think some `libkbfs` tests relies on `FakeIDByte`. I guess we should just change them to random in the future. I'll go ahead and use `tlf.MakeRandomID` in kbfs-server tests